### ISSUE TITLE
ValueError in enforce_numeric for Pandas dataframe

### ISSUE THREAD

**Issue by flacle** [RianneSchouten/pyampute#28]
Steps to reproduce.

1.  Create a new Colab notebook, paste this in a cell:
```python
!pip install pmlb==1.0.2a0
!git clone https://github.com/RianneSchouten/pyampute.git
import os
import sys
sys.path.append(os.path.join(os.getcwd(), "pyampute"))
import pmlb as PMLB
from pyampute.ampute import MultivariateAmputation
df_pmlb = PMLB.fetch_data('churn', dropna=True)
print(df_pmlb.isnull().values.any())
print(df_pmlb.info())
ma = MultivariateAmputation()
df = ma.fit_transform(df_pmlb)
print(df.isnull().values.any())
print(df.info())
```
2. Execute the cell. Error at line 51 of pyutils.py

**Comment by davzaman** [comment]
Hi @flacle, thanks for checking out our project, and apologies for the delay. I will take a look at this and get back to you within this week.

**Comment by davzaman** [comment]
Great catch, it looks like I was failing to deal with boolean column masks. While your fix in your PR (#29) was a great start, I adjusted it a bit so it would pass with our tests, and expanded our tests. I reran your code with my fix in a1621d7 and it ran without error. If you find that you rerun without error please close this issue!

Thanks again for your contribution.

**Comment by flacle** [comment]
Np, glad to help.
